src/lib.jl

```diff
@@ -516,3 +515,35 @@ Python function.
 function entropic_gromov_wasserstein(μ, ν, Cμ, Cν, ε, loss="square_loss"; kwargs...)
     return pot.gromov.entropic_gromov_wasserstein(Cμ, Cν, μ, ν, loss, ε; kwargs...)
 end
+
+"""
+    mm_unbalanced(a, b, M, reg_m; kwargs...)
+
+Solve the unbalanced optimal transport problem and return the OT plan.
+The function solves the following optimization problem:
+
+```math
+    W = \\min_\\gamma \\quad \\langle \\gamma, \\mathbf{M} \\rangle_F +
+    \\mathrm{reg_{m1}} \\cdot \\mathrm{div}(\\gamma \\mathbf{1}, \\mathbf{a}) +
+    \\mathrm{reg_{m2}} \\cdot \\mathrm{div}(\\gamma^T \\mathbf{1}, \\mathbf{b}) +
+    \\mathrm{reg} \\cdot \\mathrm{div}(\\gamma, \\mathbf{c}) \\\\
+
+    s.t.
+         \\gamma \\geq 0
+```
+
+where:
+
+- ``\\mathbf{M}`` is the (``dim_a``, ``dim_b``) metric cost matrix.
```

Comment:
`dim_a` etc. are not defined, maybe just write
```suggestion
- `M` is the metric cost matrix,
```